spec/mailers/jobapps_mailer_spec.rb

```diff
@@ -210,4 +210,61 @@
       expect(output.body.encoded).to include @user.email
     end
   end
+
+  describe 'saved_application_notification' do
+    before :each do
+      @record = create :application_record, email_to_notify: 'foo@example.com'
+    end
+    let :output do
+      JobappsMailer.saved_application_notification @record
+    end
+    it 'emails to the email_to_notify value' do
+      expect(output.to).to eql Array('foo@example.com')
+    end
+    it 'has a subject that includes the words Saved application' do
+      expect(output.subject).to include 'Saved application'
+    end
+    it 'includes the position name' do
+      expect(output.body.encoded).to include @record.position.name
+    end
+    it "includes the applicant's full name" do
+      expect(output.body.encoded).to include @record.user.full_name
+    end
+    it 'includes the date applied' do
+      expect(output.body.encoded)
+        .to include @record.created_at.strftime('%m/%d/%Y %H:%M')
+    end
+  end
+
+  describe 'saved_applications_notification' do
+    before :each do
+      record_1 = create :application_record
+      record_2 = create :application_record
+      @info = { 'Bus' => [record_1, record_2] }
+      @email = 'foo@example.com'
+    end
+    let :output do
+      JobappsMailer.saved_applications_notification @info, @email
+    end
+    it 'emails from default configured value' do
+      expect(output.to).to eql Array(@email)
+    end
+    it 'has a subject that includes the words Saved applications' do
+      expect(output.subject).to include 'Saved applications'
+    end
+    it 'includes the position names' do
+      expect(output.body.encoded).to include @info['Bus'].first.position.name
```

Comment:
Seems simpler to define the application records as instance variables and use them in your expectations here and below.